docs/notebooks/2.3 Dust Soiling with Kimber.ipynb

```diff
@@ -0,0 +1,398 @@
+   "execution_count": 4,
+   "id": "b5666543-fdd3-4359-867c-4207c7283d81",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>precipitation_rate</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>period_mid</th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>2024-05-01 04:30:00+00:00</th>\n",
+       "      <td>0.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2024-05-01 05:30:00+00:00</th>\n",
+       "      <td>0.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2024-05-01 06:30:00+00:00</th>\n",
+       "      <td>0.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2024-05-01 07:30:00+00:00</th>\n",
+       "      <td>0.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2024-05-01 08:30:00+00:00</th>\n",
+       "      <td>0.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                           precipitation_rate\n",
+       "period_mid                                   \n",
+       "2024-05-01 04:30:00+00:00                 0.0\n",
+       "2024-05-01 05:30:00+00:00                 0.0\n",
+       "2024-05-01 06:30:00+00:00                 0.0\n",
+       "2024-05-01 07:30:00+00:00                 0.0\n",
+       "2024-05-01 08:30:00+00:00                 0.0"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "rainfall_data = solcast.get_solcast_historic(\n",
+    "    latitude, longitude, \n",
+    "    start=start, end=end, \n",
+    "    api_key=os.getenv('SOLCAST_API_KEY'), \n",
+    "    output_parameters=['precipitation_rate'],\n",
+    "    period=\"PT1H\"\n",
+    ")[0]\n",
+    "\n",
+    "rainfall_data.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e14906a4-5289-4e81-ad54-f0ac02184edc",
+   "metadata": {},
+   "source": [
+    "We create a new column `accumulated_rainfall` with the daily total rainfall, i.e. the total rainfall in the preceding 24h. This is the same that the Kimber algorithm does: when this accumulation reaches the cleaning threshold, the soiling rate returns to 0."
```

Comment:
We should make it clear that the accumulated rainfall is for interpretation, not for input to pvlib. Suggest

"""
To assist in the interpretation of results, we create a new column `accumulated_rainfall` with the daily total rainfall, i.e. the total rainfall in the preceding 24h. This matches what the Kimber algorithm does: when this accumulation reaches the cleaning threshold, the soiling rate returns to 0.

We also set up a plotting function to superimpose the accumulation on the soiling rate results.
"""